aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -201,23 +201,23 @@ void dirichlet_scalar_cuda_kernel(
 namespace at { namespace native {
 Tensor _s_poisson_cuda(const Tensor& lambda, Generator* gen) {
   Tensor ret = at::empty(lambda.sizes(), lambda.options());
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(ret.scalar_type(), "poisson_cuda", [&] {
+  AT_DISPATCH_ALL_TYPES_AND(ret.scalar_type(), "poisson_cuda", [&] {
```

Comment:
This seems to be the cause of build errors, looks like it needs to be 
```
AT_DISPATCH_ALL_TYPES_AND(at::ScalarType::Half, ret.scalar_type(), "poisson_cuda", [&] {
```
(see [here](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/Dispatch.h#L52)).

Reply:
Whoops, I missed a few. Too late at night? Will fix and re-push.

Reply:
Umm, this file shouldn't change at all on this PR, right? Maybe a merge with master is in order.

Reply:
@ezyang I merged with master yesterday, this still seems to be the error? I can merge again.

Reply:
@anirrudh you made this change in commit 5c3fd777d8eab15. It's a fix for a deprecation, but the fix is incomplete. If @ezyang says it should not be part of this PR, just revert that commit?

Reply:
Oh, my apologies. I will revert that commit -- I misunderstood!